regression/strings-smoke-tests/java_append_int/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+FUTURE
```

Comment:
These tests have never actually been enabled, because the test suite was not called from the `regression/Makefile`. For this one in particular, the behaviour might be affected when we add models for strings. It is just easier to go through all these tests once everything is merged, then enable those that work, and create issues for those that don't. There is no particular issue to enable the tests, but it is an ongoing process.